<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Формирование отчета по успеваемости</string>
  </property>
  <widget class="Line" name="line_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>600</width>
     <height>16</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>600</width>
     <height>20</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_zachto">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>200</y>
     <width>215</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_student">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>82</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string>Ученик:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_students">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>80</y>
     <width>215</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_subject">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>122</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string>Предмет:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_subject">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>120</y>
     <width>215</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_zachto">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>202</y>
     <width>135</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string>Тип задания:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_date">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>33</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string>Дата:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_grade">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>161</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string>Оценка:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_grade">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>160</y>
     <width>215</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_writeToDB">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>90</y>
     <width>201</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(253, 142, 15);</string>
   </property>
   <property name="text">
    <string>Записать в БД</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>600</width>
     <height>20</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>40</y>
     <width>215</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="Russian" country="Russia"/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="currentSectionIndex">
    <number>0</number>
   </property>
   <property name="date">
    <date>
     <year>2023</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>258</y>
     <width>601</width>
     <height>192</height>
    </rect>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>№</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Дата</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Ученик</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Предмет</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Оценка</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Тип задания</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
